modules/integration_azure-container-instance/conf/readme.yaml

```diff
@@ -0,0 +1,3 @@
+documentations:
+
+source_doc:
```

Comment:
may be add `notes` to warn about the risk to get false alert for "temp"/"short lived" containers ?

Reply:
done